### ISSUE TITLE
Envenom modification

### ISSUE THREAD

**Issue by Cluetoris** [tautges/runelite-poisoned-npcs-plugin#2]
I was hoping to be able to modify this for a envenom mechanics as well,

**Comment by tautges** [comment]
Cool! Were you wanting to make the modifications yourself and submit a PR, looking for guidance on how to add them, or making a feature request for that to be added?

**Comment by Cluetoris** [comment]
Yes to any, i think it would be fun to do it myself but i have 0 experience
coding. i dont know what im doing though i downloaded the jdk 11 and
intellij, i got to your source code. I used to packet edit a long time ago
so leeching and modifying was my thing lol
